GANDLF/losses/__init__.py

```diff
@@ -38,3 +38,26 @@
     "focal": FocalLoss,
     "dc_focal": DC_Focal,
 }
+
+
+def get_loss(params):
```

Comment:
What should the return type be?

```suggestion
def get_loss(params: dict):
```

Reply:
Will add